controllers/extension-certificate-service/charts/cert-broker-rbac/templates/rbac.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: gardener.cloud:system:cert-broker
```

Comment:
In gardener, this is called `garden.sapcloud.io:system:cert-broker`. Should we keep the name for now?